dominion/templates/_base.html

```diff
@@ -11,6 +11,9 @@
 {% endblock %}
 
 {% block body_javascript %}
+  <script src="{% static asset_manifest.runtime %}"></script>
+  <script src="{% static asset_manifest.vendor %}"></script>
+  <script src="{% static asset_manifest.main %}"></script>
```

Comment:
The reason I usually put my custom scripts **after** a call to `{{ block.super }}` is to make sure if there is any name clash, my scripts win out.